plugin/backend.go

```diff
@@ -22,7 +22,12 @@ import (
 	"google.golang.org/api/option"
 )
 
-const userAgentPluginName = "auth-gcp"
+const (
+	userAgentPluginName = "auth-gcp"
+
+	// operationPrefixGoogleCloud is used as a prefix for OpenAPI operation id's.
+	operationPrefixGoogleCloud = "google-cloud"
```

Comment:
Is this the consistent naming throughout? vs `GCP`

Reply:
This is how the plugin is referenced to in the [docs](https://developer.hashicorp.com/vault/api-docs/secret/gcp). I felt `gcp` might be too obscure but I'm happy to change this if folks feel strongly about it.

Reply:
I don't feel too strongly either way but maybe it would make sense to use `gcp` since (1) that is how we reference it everywhere except for the docs and (2) the operationID is an ID so I think the only thing that really matters is uniqueness.

Reply:
> (2) the operationID is an ID so I think the only thing that really matters is uniqueness.

The `operationID` is translated directly into method names and struct names in the [library generated code](https://github.com/hashicorp/vault-client-go/blob/main/api_auth.go#L3670-L3671). For example `google-cloud-list-roles` would be translated into:

```
func (a *Auth) GoogleCloudListRoles(ctx context.Context, options ...RequestOption) (*GoogleCloudListRolesResponse, error) {
    ...
}
```

vs

```
func (s *Secrets) GcpListRoles(ctx context.Context) (*GcpListRolesResponse, error) {
    ...
}
```

Reply:
Got it, thanks!  google-cloud works for me!